bundles/org.openhab.binding.androidtv/README.md

```diff
@@ -30,37 +32,53 @@ This binding requires GoogleTV to be installed on the device (https://play.googl
 | ipAddress        | text    | IP address of the device              | N/A     | yes      | no       |
 | googletvPort     | text    | TCP Port for GoogleTV                 | 6466    | no       | no       |
 | shieldtvPort     | text    | TCP Port for ShieldTV                 | 8987    | no       | no       |
+| philipstvPort    | text    | TCP Port for PhilipsTV                | 1926    | no       | no       |
 | keystore         | text    | Location of the Java Keystore         | N/A     | no       | no       |
 | keystorePassword | text    | Password of the Java Keystore         | N/A     | no       | no       |
 | gtvEnabled       | boolean | Enable/Disable the GoogleTV protocol  | true    | no       | no       |
 
 ```java
 Thing androidtv:shieldtv:livingroom [ ipAddress="192.168.1.2" ]
 Thing androidtv:googletv:theater [ ipAddress="192.168.1.3" ]
+Thing androidtv:philipstv:bedroom [ ipAddress="192.168.1.4" ]
 ```
 
 ## Channels
 
-| Channel    | Type   | Description                 | GoogleTV | ShieldTV |
-|------------|--------|-----------------------------|----------|----------|
-| keyboard   | String | Keyboard Data Entry         |    RW    |    RW    | 
-| keypress   | String | Manual Key Press Entry      |    RW    |    RW    |
-| keycode    | String | Direct KEYCODE Entry        |    RW    |    RW    |
-| pincode    | String | PIN Code Entry              |    RW    |    RW    |
-| app        | String | App Control                 |    RO    |    RW    |
-| appname    | String | App Name                    |    N/A   |    RW    |
-| appurl     | String | App URL                     |    N/A   |    RW    |
-| player     | Player | Player Control              |    RW    |    RW    |
-| power      | Switch | Power Control               |    RW    |    RW    |
-| volume     | Dimmer | Volume Control              |    RO    |    RO    |
-| mute       | Switch | Mute Control                |    RW    |    RW    |
+| Channel              | Type   | Description                          | GoogleTV | ShieldTV | PhilipsTV |
+|----------------------|--------|--------------------------------------|----------|----------|-----------|
+| keyboard             | String | Keyboard Data Entry                  |    RW    |    RW    |     RW    |
+| keypress             | String | Manual Key Press Entry               |    RW    |    RW    |     RW    |
+| keycode              | String | Direct KEYCODE Entry                 |    RW    |    RW    |     RW    |
+| pincode              | String | PIN Code Entry                       |    RW    |    RW    |     RW    |
+| app                  | String | App Control                          |    RO    |    RW    |     RW    |
+| appname              | String | App Name                             |    N/A   |    RW    |     RW    |
+| appurl               | String | App URL                              |    N/A   |    RO    |     N/A   |
```

Comment:
Is it intentionall that this the `appurl` for the `ShieldTv` is changed from RW to RO?

Reply:
I don't particularly remember making the change in this file for it, but appurl has always been RO on the shield so RO is accurate either way.